File: multi-ads-mediator-sdk/src/main/java/com/mdkhaki/multi_ads_mediator_sdk/NetworkCustom.java
package com.mdkhaki.multi_ads_mediator_sdk;

public class NetworkCustom {
    public boolean tapsellAds = false;
    public boolean unityAds = false;
    public boolean chartBoostAds = false;

    public NetworkCustom(boolean tapsellAds, boolean unityAds, boolean chartBoostAds) {
        this.tapsellAds = tapsellAds;
        this.unityAds = unityAds;
        this.chartBoostAds = chartBoostAds;
    }

    public boolean isTapsellAds() {
        return tapsellAds;
    }

    public void setTapsellAds(boolean tapsellAds) {
        this.tapsellAds = tapsellAds;
    }

    public boolean isUnityAds() {
        return unityAds;
    }

    public void setUnityAds(boolean unityAds) {
        this.unityAds = unityAds;
    }

    public boolean isChartBoostAds() {
        return chartBoostAds;
    }

    public void setChartBoostAds(boolean chartBoostAds) {
        this.chartBoostAds = chartBoostAds;
    }
}

File: multi-ads-mediator-sdk/src/main/java/com/mdkhaki/multi_ads_mediator_sdk/NetworkCustom.kt
package com.mdkhaki.multi_ads_mediator_sdk

class NetworkCustom(tapsellAds: Boolean = true, unityAds: Boolean = true, chartBoostAds: Boolean = true) {
    var tapsellAds = false
    var unityAds = false
    var chartBoostAds = false

    init {
        this.tapsellAds = tapsellAds
        this.unityAds = unityAds
        this.chartBoostAds = chartBoostAds
    }
}